VideoLocker/res/drawable/row_discussion_first_comment_background.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+    <solid android:color="@color/edx_grayscale_neutral_white" />
+    <stroke
+        android:width="@dimen/edx_hairline"
+        android:color="@color/edx_grayscale_neutral_light" />
+    <corners android:radius="@dimen/edx_box_radius"
```

Comment:
Looks like this issue has been fixes in API 12, so we shouldn’t have to worry about it: https://stackoverflow.com/a/6376398
> On 08-Sep-2015, at 8:01 pm, Brian Guertin <notifications@github.com> wrote:
> 
> In VideoLocker/res/drawable/row_discussion_first_comment_background.xml <https://github.com/edx/edx-app-android/pull/348#discussion_r38936071>:
> 
> > @@ -0,0 +1,13 @@
> > +<?xml version="1.0" encoding="utf-8"?>
> > +<shape xmlns:android="http://schemas.android.com/apk/res/android"
> > +    android:shape="rectangle">
> > +    <solid android:color="@color/edx_grayscale_neutral_white" />
> > +    <stroke
> > +        android:width="@dimen/edx_hairline"
> > +        android:color="@color/edx_grayscale_neutral_light" />
> > +    <corners android:radius="@dimen/edx_box_radius"
> It's per the Android docs, I don't know what API levels are affected:
> "Note: Every corner must (initially) be provided a corner radius greater than 1, or else no corners are rounded. If you want specific corners to not be rounded, a work-around is to use android:radius to set a default corner radius greater than 1, but then override each and every corner with the values you really want, providing zero ("0dp") where you don't want rounded corners."
> 
> http://developer.android.com/guide/topics/resources/drawable-resource.html#Shape <http://developer.android.com/guide/topics/resources/drawable-resource.html#Shape>
> —
> Reply to this email directly or view it on GitHub <https://github.com/edx/edx-app-android/pull/348/files#r38936071>.
>